xml/signals/AAR-1946/appearance-SL-2-high-pbs.xml

```diff
@@ -49,7 +49,8 @@
       <revnumber>2.3</revnumber>
       <date>2019-12-30</date>
       <authorinitials>BM</authorinitials>
-      <revremark>Remove ourAspect values which are not supported by this mast type</revremark>
+      <revremark>Removed ourAspect values which are not supported by this mast 
+          type. Add missing "advancedAspect" mappings.</revremark>
```

Comment:
Shouldn't a new revision 2.4 be added here?

Reply:
Absolutely!

Reply:
Resolved.